tileview/src/main/java/com/qozix/tileview/tiles/TileCanvasViewGroup.java

```diff
@@ -242,14 +243,19 @@ void onRenderTaskCancelled() {
 
   void onRenderTaskPostExecute() {
     mIsRendering = false;
-    if( !mTransitionsEnabled ) {
-      cleanup();
-    }
-    if( mTileRenderListener != null ) {
-      mTileRenderListener.onRenderComplete();
-    }
-    invalidate();
-    requestRender();
+    mTileRenderHandler.post( new Runnable() {
```

Comment:
We are reaching this portion of code from multiple threads (not from ui thread).
From doc of the post method :

> Causes the Runnable r to be added to the message queue. The runnable will be run on the thread to which this handler is attached.

In this case, the handler is attached to the ui thread. So i would say that it's ok to post the same instance of `Runnable`, because then no mutual exclusion is need to protect us from accessing ui object from multiple threads.